src/pages/AdminConsole.js

```diff
@@ -177,7 +177,7 @@ class AdminConsole extends Component {
                 }),
               ]),
               div({
-                isRendered: (this.props.env !== 'prod'),
+                isRendered: this.state.showAddUserModal,
```

Comment:
This div should always be enabled now. It is essentially a replacement for the previous div.